followthemoney/compare.py

```diff
@@ -26,22 +31,22 @@
 }
 
 
-def compare(model, left, right):
+def compare(model: Model, left: Mapping, right: Mapping) -> float:
```

Comment:
... Union[Mapping,EntityProxy]

Reply:
Would it make sense to make `EntityProxy` inherit on `Mapping` then? because right now, it is not possible to call `dict(EntityProxy)`:
https://github.com/alephdata/followthemoney/blob/91e56b450a9c1bc29221e40cfc2512dfd1f97dfd/followthemoney/proxy.py#L25

Reply:
... or at least I'm super confused how that is supposed to work right now.

Based on the doc, I cannot see how you can pass an `EntityProxy` to `dict` -> https://docs.python.org/3.8/library/stdtypes.html#dict